### ISSUE TITLE
Line wrapping

### ISSUE THREAD

**Issue by GoldenGlimpse** [FFDA/SourCherry#22]
Any possibility to enable/disable line wrapping? 

Thank you.

**Comment by FFDA** [comment]
No, there isn’t currently an option to do this. It took me some time to solve a visual glitch, but I believe it’s possible to implement.

To make sure I understand what you’re expecting, I made a short video. So, instead of wrapping the text on all lines to match the width of the screen, you’d like the text to look like this:

https://github.com/user-attachments/assets/e53b1a2d-4e7c-4518-b3e8-7715d372e50f

**Comment by GoldenGlimpse** [comment]
Yes, exactly. For me, Sourcherry is unusable on Android without the line wrapping.

Unfortunately, i had to switch to a different note taking app already. Migrated the whole thing.
I'll keep an eye open for improvements on the cherrytree/sourcherry project.
I really liked my cherries...

**Comment by FFDA** [comment]
OK, I’m closing the issue as it’s no longer relevant.
